cli/display.cpp

```diff
@@ -0,0 +1,72 @@
+#include <exception>
+#include <stdexcept>
+#include <string>
+#include "display.h"
+
+const int displayCount() {
+    int i = 0;
+    while (1) {
+        DEVMODE dm;
+        DISPLAY_DEVICE dd;
+        dd.cb = { sizeof(DISPLAY_DEVICE) };
+        if (!EnumDisplayDevices(NULL, i++, &dd, 0) 
```

Comment:
```suggestion
        if (!EnumDisplayDevices(nullptr, i++, &dd, 0) 
```

Reply:
you could just use `i` here and then increase it after the `if` which would remove the necessity to do `return i-1;`